### ISSUE TITLE
[FE] My VA: Simplify Cerner alert

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#28575]
## Background
As noted in #25049 the Cerner alert needs to be simplified. We submitted a [content ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/28395) for assistance in reviewing the content and design and have a new design that needs to be implemented. 

Feedback from content documented [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25049#issuecomment-894440005). We're going with version 4 outlined below:

_# V4_

## Choose your health management portal

Your care team may now use our new My VA Health portal. Choose your portal based on the facility for your appointment:

- **For Charles P. Wylie Veterans Outpatient Clinic:** [link] Use My VA Health
- **For any other VA health facility:** [link] Use My HealtheVet

**If you have trouble accessing the portal:** Make sure to disable your pop-up blocker. If you’re prompted to sign in again, use the same account you used to sign in to VA.gov.

## Screenshots
![Capture](https://user-images.githubusercontent.com/84803567/129403400-4530a520-ba93-4ee9-91fb-f465a9cb2da3.PNG)

## Tasks
- [x] Update design/copy for Cerner alert on My VA
- [x] Ensure proper styling of header and links

**Comment by tmitchellgcio** [comment]
@tressaellen This is what I have currently, however the image you have doesn't have the grey background and it looks more stretched out. Considering the content on the page, if I stretched it out it may look off. See the image below. Thoughts? 

![Cerner Alert](https://user-images.githubusercontent.com/84803567/129288940-e6744c7c-0249-4e6b-8885-b11ae69bbdf7.PNG)
![Comparison to content](https://user-images.githubusercontent.com/84803567/129291176-502051a8-233f-40f6-9306-b2e5bb5edee5.PNG)

**Comment by tressaellen** [comment]
Hi @tmitchellgcio - yes, it does need the grey background. And go ahead and just use the default width of the component - we aren't changing the width of any of the other errors so we'll just leave it for consistency. 

@DanielleThierryUSDSVA - there is so much bold going on in this alert that it's difficult to know what is the most important part. Could we change the **"If you have trouble accessing the portal:"** to "If you have trouble accessing the portal, make sure to..." (Remove the bold and the colon and make it a continuous sentence- shown in screenshot)

<img width="813" alt="Screen Shot 2021-08-13 at 9 41 12 AM" src="https://user-images.githubusercontent.com/58752772/129398516-d5c5ca56-f4dc-445e-a50c-a04d07362217.png">

**Comment by tmitchellgcio** [comment]
> Hi @tmitchellgcio - yes, it does need the grey background. And go ahead and just use the default width of the component - we aren't changing the width of any of the other errors so we'll just leave it for consistency.
> 
> @DanielleThierryUSDSVA - there is so much bold going on in this alert that it's difficult to know what is the most important part. Could we change the **"If you have trouble accessing the portal:"** to "If you have trouble accessing the portal, make sure to..." (Remove the bold and the colon and make it a continuous sentence- shown in screenshot)
> 
> <img alt="Screen Shot 2021-08-13 at 9 41 12 AM" width="813" src="https://user-images.githubusercontent.com/58752772/129398516-d5c5ca56-f4dc-445e-a50c-a04d07362217.png">

Thank you!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @tressaellen. I think that's fine. Thanks for checking!

**Comment by tressaellen** [comment]
@DanielleThierryUSDSVA thank you!

@tmitchellgcio - let's go ahead and change the copy to the following (View screenshot above):

Choose your health management portal

Your care team may now use our new My VA Health portal. Choose your portal based on the facility for your appointment:

For **Charles P. Wylie Veterans Outpatient Clinic** or **Mann-Grandstaff Department of Veterans Affairs Medical Center**: [link] Use My VA Health

For **any other VA health facility:** [Use My HealtheVet]

If you have trouble accessing the portal, make sure to disable your pop-up blocker. If you're prompted to sign in again, use the same account you used to sign in to VA.gov.

**Comment by tmitchellgcio** [comment]
![Capture](https://user-images.githubusercontent.com/84803567/129774514-4a049cb3-cd9c-4d48-a364-0e0ef8952108.PNG)
This is the updated screenshot for the newest PR. Let me know if there are any other changes. If not then I will merge the PR. Thank you @tressaellen

**Comment by ajakabcin** [comment]
Is "For" in "For any other VA health facility:.." supposed to be bolded like the rest of the phrase? @tressaellen and @tmitchellgcio

**Comment by tmitchellgcio** [comment]
> Is "For" in "For any other VA health facility:.." supposed to be bolded like the rest of the phrase? @tressaellen and @tmitchellgcio

I think that is supposed to not be bold like the phrase above it. That's at least what I gathered from her mock above. Open to changing it if need be though.

**Comment by ajakabcin** [comment]
Confirmed on staging with user ssoissoetesting+mhvcss4@gmail.com
